source/extensions/tracers/opentelemetry/samplers/sampler.h

```diff
@@ -25,9 +25,9 @@ enum class Decision {
   // dropped.
   DROP,
   // IsRecording will be true, but the Sampled flag MUST NOT be set.
-  RECORD_ONLY,
```

Comment:
not sure if `DROP` should also be changed - it wasnt flagged by clang-tidy for some reason

Reply:
I think it should be updated to `Drop`. :)